DevGuideExamples/DCPS/Messenger/Publisher.cpp

```diff
@@ -78,21 +82,26 @@ ACE_TMAIN(int argc, ACE_TCHAR *argv[])
       ACE_ERROR_RETURN((LM_ERROR,
                         ACE_TEXT("ERROR: %N:%l: main() -")
                         ACE_TEXT(" create_publisher failed!\n")),
-                       -1);
+                       1);
     }
 
     // Create DataWriter
+    DDS::DataWriterQos writer_qos;
+    publisher->get_default_datawriter_qos(writer_qos);
+    writer_qos.reliability.kind = DDS::RELIABLE_RELIABILITY_QOS;
+    writer_qos.history.kind = DDS::KEEP_ALL_HISTORY_QOS;
```

Comment:
This is an issue with the "minimum profile".  We discussed a bit, but just recording it here.